### DISCUSSION TITLE
Redundant rules between this package and a11y

### DISCUSSION THREAD

**Discussion by sacrosanctic** [sveltejs/eslint-plugin-svelte#639] [General]
What is the purpose of a rule like [svelte/no-target-blank](https://sveltejs.github.io/eslint-plugin-svelte/rules/no-target-blank/) when a11y (which is built into the svelte compiler) already warns you?

**Comment by ota-meshi** [comment]
We implemented this rule before the Svelte compiler started warning about it.

**Reply by sacrosanctic** [reply]
Any plans on depreciating the rule or adding a note about similar functionality in the docs?

**Reply by baseballyama** [reply]
Maybe can remove this from recommended config when we define like `plugin:svelte/svelte5-recommended`.
